scripts/preinstall.js

```diff
@@ -16,7 +16,7 @@ const CLIENTS_DIR = path.resolve(BPANEL_DIR, 'clients');
 const LOCAL_PLUGINS_DIR = path.resolve(BPANEL_DIR, 'local_plugins');
 
 const configText = `module.exports = {
-  plugins: ['@bpanel/genesis-theme', '@bpanel/bui'],
+  plugins: ['@bpanel/genesis-theme', '@bpanel/bui', '@bpanel/settings', '@bpanel/connection-manager'],
```

Comment:
What do you think about removing `bui` as a default plugin at this point? Now that we have some actual utility, it doesn't make sense to me as much to include it

Reply:
Agree with removing `bui` -- should this change also be applied in `docker-compose.yml` ?

Reply:
I agree with removing it from `docker-compose` as well

Reply:
Agreement all around! Will update accordingly.